style.css

```diff
@@ -0,0 +1,481 @@
+/* stylelint-disable */
+
+
+/* STYLE SHEET */
+
+
+/* GLOBALS */
+
+body {
+    max-width: 144rem;
+    margin: 0 auto;
```

Comment:
You are using this property 6 times throughout your CSS, maybe it would be better to make an `m-0` class? :wink:

Reply:
fixed